modular_skyrat/master_files/code/modules/mob/living/simple_animal/friendly/dogs.dm

```diff
@@ -145,7 +160,7 @@
 		do_sparks(3, 1, src)
 
 /mob/living/simple_animal/pet/dog/corgi/borgi/handle_automated_action()
-	if(emagged && prob(25))
+	if(emagged && prob(50))
```

Comment:
why is this being raised?

Reply:
I didn't think it ticked enough before I even got the idea to make this PR, but I understand it may be too high. I'll put it on 33 because I really like that number.

Reply:
Sure